browser.js

```diff
@@ -56,11 +88,16 @@ ECDH.prototype.setPrivateKey = function (priv, enc) {
 	_priv = _priv.toString(16);
 	this.keys._importPrivate(_priv);
 };
-function returnValue(bn, enc) {
+function returnValue(bn, enc, len) {
```

Comment:
`returnValue` is a bit lame.  It looks like this code is padding to a given length and then returning the buffer in a particular encoding.

Can we make this into 2 separate steps?